utils/oc.py

```diff
@@ -212,8 +212,6 @@ def recycle_pods(self, dry_run, namespace, dep_kind, dep_resource):
         qontract_recycle = dep_annotations.get('qontract.recycle')
         if qontract_recycle is True:
             raise RecyclePodsInvalidAnnotationValue('should be "true"')
-        if not isinstance(qontract_recycle, str):
-            raise RecyclePodsInvalidAnnotationValue('should be a string')
```

Comment:
Yeah.. python was born to be duck-typed... What about one check to rule them all?
```python
qontract_recycle = dep_annotations.get('qontract.recycle')
if str(qontract_recycle).lower() != 'true':
    return
```

Reply:
That's more or less what we had before. I think @maorfr wanted to make sure we warn people that they should not use the boolean `true` but the string `"true"`. He did that in https://github.com/app-sre/qontract-reconcile/pull/874 but went too far 😬 

I like the idea of being nice and warning people as much as possible

Reply:
yes..but before we had no handler for `True`... and if users said `qontract.recycle: true` it is quite clear that they mean the recycle to happen. Enforcing types is a waste of tentant cycles imo, but I'm not super strict here :)

Reply:
I don't know about the reasons to make it a string, honestly. But he went quite far to make sure it is, so I'd rather keep it that way until he can give more context :-D

Reply:
a ConfigMap will also fail to apply if an annotation value is boolean IIRC

Reply:
then forget what I said :)